plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/functions/tablechanges/TableChangesFunction.java

```diff
@@ -74,7 +78,8 @@ public TableChangesFunction(DeltaLakeMetadataFactory deltaLakeMetadataFactory)
                 ImmutableList.of(
                         ScalarArgumentSpecification.builder().name(SCHEMA_NAME_ARGUMENT).type(VARCHAR).build(),
                         ScalarArgumentSpecification.builder().name(TABLE_NAME_ARGUMENT).type(VARCHAR).build(),
-                        ScalarArgumentSpecification.builder().name(SINCE_VERSION_ARGUMENT).type(BIGINT).defaultValue(null).build()),
+                        ScalarArgumentSpecification.builder().name(SINCE_VERSION_ARGUMENT).type(BIGINT).defaultValue(DEFAULT_GUARD_VALUE).build(),
+                        ScalarArgumentSpecification.builder().name(SINCE_TIMESTAMP_ARGUMENT).type(TIMESTAMP_MILLIS).defaultValue(null).build()),
```

Comment:
TIMESTAMP_MILLIS is a wrong type for this, it does not represent point in time.

use TIMESTAMP_TZ_MILLIS